jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -193,9 +193,10 @@ def _on_viewer_data_changed(self, msg=None):
         self._bounds["max"] = viewer_data.spectral_axis[-1]
 
         # set redshift slider to redshift stored in Spectrum1D object
-        self.rs_redshift = (viewer_data.redshift.value
-                            if hasattr(viewer_data.redshift, 'value')
-                            else viewer_data.redshift)
+        if viewer_data.meta.get('plugin', None) is not None:
+            self.rs_redshift = (viewer_data.redshift.value
```

Comment:
How come it says this is not covered in test?

Reply:
I'm guessing we don't have any tests that use line lists and then another plugin.  I can write a regression test to add coverage if we want, but it would be quite particular to just cover this specific line.

Reply:
If adding a test isn't too hard, please add it. Having even a very specific test would prevent the bug from creeping back in. Thanks!

Reply:
test added